packages/api/.env

```diff
@@ -9,3 +9,5 @@ DASHCORE_USER=username
 DASHCORE_PASS=password
 EPOCH_CHANGE_TIME=3600000
 DAPI_URL=127.0.0.1:1443:self-signed
+RATER_URLS='https://api.kucoin.com/api/v1/market/stats?symbol=DASH-USD, https://api.coinbase.com/v2/exchange-rates?currency=DASH'
```

Comment:
There is no need to specify rates url through the environment, because you can't set there anything except those you have implemented in the app